### PR TITLE
homework_3

### PR DESCRIPTION
**Pull request by junyaa**

### CODE REVIEW

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+function square(a) {
+    return (a ** 2);
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1.js]: the condition is not to use `**` operator

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1_3.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+function createarray(a, b) {
+    let i;
+    const arr = [];
+    for (i = 0; i <= b; i += a) {
+        arr.push(i ** 2);
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1_3.js]: same, do not use `**` operator pls

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1_2.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+function createarray(a, b) {
+    let i;
+    const arr = [];
+    for (i = 0; i <= b; i += a) {
+        arr.push(i);
+    }
+    console.log(arr);
+}
+createarray(4, 32); // (step, max)
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1_2.js]: good

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+function func(x) {
+    if (typeof x !== 'number') {
+        return 'Not a number';
+    }
+    return (Math.log(x) / Math.log(2)) % 1 === 0;
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js]: the condition of task is not to use `Math` methods

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1.js (hunk 1 above)
Comment (junyaa) [reply homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1.js]: Но если делать через Math.pow, то линтер рекомендует именно **
 error  'Math.pow' is restricted from being used. Use the exponentiation operator (**) instead

Comment (junyaa) [reply homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1.js]: > the condition is not to use `**` operator

fixed

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1_3.js (hunk 2 above)
Comment (junyaa) [reply homeworks/yuriy.piddubnyak_junyaa/homework_3/task_1_3.js]: > same, do not use `**` operator pls

fix

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js (hunk 4 above)
Comment (junyaa) [reply homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js]: > the condition of task is not to use `Math` methods

fixed

File: homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+function isPower(v) {
+    if (typeof v !== 'number') {
+        return 'Not a number';
+    }
+    if (v == 1) {
```
Comment (markoffden) [inline homeworks/yuriy.piddubnyak_junyaa/homework_3/task_2.js]: this will include 1 in the set

**Pull request merged**